### PR TITLE
Fix days difference counting in the schedule

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1625

### CODE REVIEW

File: src/hooks/useVenueScheduleEvents.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import { useRelatedVenues } from "./useRelatedVenues";
 
 const emptyRelatedEvents: WithVenueId<VenueEvent>[] = [];
 const minRangeValue = 0;
-const todaysDate = new Date();
+const todaysDate = startOfToday();
```
Comment (sunny-viktoryia) [inline src/hooks/useVenueScheduleEvents.ts:28]: this is the fix

Comment (goran-peoski-work) [reply src/hooks/useVenueScheduleEvents.ts:28]: I assume this one uses local browser TZ

File: src/hooks/useVenueScheduleEvents.ts [hunk 2]
```diff
@@ -114,7 +110,8 @@ const useVenueScheduleEvents = ({
     fromUnixTime(firstRangeDateInSeconds)
   );
 
-  const dayDifference = getEventDayRange(daysInBetween, isOneEventAndLive);
+  // add 1 day to daysInBetween to form a full timeline
```
Comment (goran-peoski-work) [inline src/hooks/useVenueScheduleEvents.ts:113]: This is just English for the line under, adds no more meaning. Would be nice to instead have the reason of why there is a need to add a day.

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/hooks/useVenueScheduleEvents.ts (hunk 1 above)
Comment (sunny-viktoryia) [reply src/hooks/useVenueScheduleEvents.ts:28]: yes, it uses a local browser timezone.

File: src/hooks/useVenueScheduleEvents.ts (hunk 2 above)
Comment (sunny-viktoryia) [reply src/hooks/useVenueScheduleEvents.ts]: changed the comment: https://github.com/sparkletown/sparkle/pull/2720/commits/658dc03ac29ab71d1ea74795e3bd5e23be09a05e

**Pull request merged**